src/Table/SynapseTableReflection.php

```diff
@@ -205,4 +205,38 @@ public function isTemporary(): bool
     {
         return $this->isTemporary;
     }
+
+    /**
+     * @return array{
+     *  schema_name: string,
+     *  name: string
+     * }[]
+     */
+    public function getDependentViews(): array
+    {
+        $sql = <<<EOT
```

Comment:
Mini věc je když nepotřebuješ interpolaci tak místo heredoc možeš použit nowdoc https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.nowdoc